py3status/modules/do_not_disturb.py

```diff
@@ -10,7 +10,7 @@
     pause: specify whether to pause or kill processes; for dunst
         see `Dunst Miscellaneous` section for more information
         (default False)
-    server: specify server to use, eg dunst or xfce4-notifyd, otherwise auto
+    server: specify server to use, eg dunst, xfce4-notifyd or mako, otherwise auto
```

Comment:
If you have `mako` and `dunst` installed, but are on `sway`, this would pick `dunst`. I wonder if mako should be checked first before dunst. `xfce4-notifyd` (or any other non-i3/sway notification servers) should always be last anyway.